sdk/containerregistry/azure-containerregistry/azure/containerregistry/_exchange_client.py

```diff
@@ -76,7 +76,7 @@ def get_refresh_token(self, service, **kwargs):
 
     def exchange_aad_token_for_refresh_token(self, service, **kwargs):
         # type: (str, Any) -> str
-        refresh_token = self._client.authentication.exchange_aad_access_token_for_acr_refresh_token(
+        refresh_token = self._client.authentication.exchange_aad_access_token_for_acr_refresh_token( # type: ignore
```

Comment:
This ignore looks like a real bug to me -- it doesn't look like these methods generate on the AuthenticationOperations anymore:

https://github.com/Azure/azure-sdk-for-python/blob/733a151cd13549070d64e2e3846de467fca7766b/sdk/containerregistry/azure-containerregistry/azure/containerregistry/_generated/operations/_operations.py#L2521-L2545

Reply:
Yes, there are two operations not generated, I pasted the code from previous generated to _patch.py, which is the recommended way to fix the implementation error.

Reply:
Got it, in this case you might have to run the autorest [postprocessing](https://github.com/Azure/autorest.python/blob/main/docs/customizations.md#postprocessing) script to make mypy happy since it seems like it's seeing AuthenticationOperations as imported from the generated code. cc/ @iscai-msft to confirm.

Reply:
Update: ignore the mypy error for now, the script failed to get `azure.containerregistry._patch.py`